src/main/kotlin/org/pkl/lsp/ast/PklNode.kt

```diff
@@ -717,7 +717,7 @@ abstract class AbstractPklNode(
 
   override val children: List<PklNode> by lazy { childrenByType.values.flatten() }
 
-  override val text: String by lazy { ctx.utfAwareText() }
+  override val text: String by lazy { ctx.text ?: "" }
```

Comment:
Shouldn't the `null` case still `throw`?

Reply:
In practice this won't ever happen because we always create trees via a source String. But yes, in case this is ever `null` it's better to throw. I removed the default so it will null-pointer.